community_images/fluent-bit/ironbank/overrides.yml

```diff
@@ -7,8 +7,6 @@ containerSecurityContext:
   allowPrivilegeEscalation: true
   capabilities:
     add: ["SYS_PTRACE"]
-resourceType: deployment
```

Comment:
Any specific reason for removing this?

Reply:
The official helm_chart required replica_count 1 for deployment resource type but not generally required

Reply:
Can we just keep it then?

Reply:
No need as I tried using other helm chart where replica count was already there as 1 for deployment type in values.yml